README.md

```diff
@@ -147,6 +148,13 @@ This class extends [`EventEmitter`][] from Node.js.
     handling I/O in parallel.
   * `useAtomics`: (`boolean`) Use the [`Atomics`][] API for faster communication
     between threads. This is on by default.
+  * `resourceLimits`: (`object`)
```

Comment:
Maybe link to https://nodejs.org/api/worker_threads.html#worker_threads_new_worker_filename_options here? :)